File: ClientApp/src/app/models/movie.model.ts
export class ImdbSearchFilterModel {
    name: string;
    searchType: number;
    year: number;
    page: number;
}

export class ImdbSearchResultModel {
    search: ImdbSearchMovieResultModel[];
    totalResults: number;
}

export class ImdbSearchMovieResultModel {
    title: string;
    year: number;
    imdbID: string;
    type: string;
    poster: string;
}

export class MovieModel {
    title: string;
    year: string;
    rated: string;
    released: string;
    runtime: string;
    genre: string;
    director: string;
    writer: string;
    actors: string;
    plot: string;
    language: string;
    country: string;
    awards: string;
    poster: string;
    metascore: string;
    imdbRating: string;
    imdbVotes: string;
    imdbID: string;
    type: string;
    dVD: string;
    boxOffice: string;
    production: string;
    website: string;
    response: string;
}

File: Models/Movie/Movie/MovieModel.cs
namespace AllInOne.Models.Movie.Movie
{
    public class MovieModel
    {
        public long Id { get; set; }
        public string Title { get; set; }
        public string ImdbRating { get; set; }
        public string Year { get; set; }
        public string Rated { get; set; }
        public string Genre { get; set; }
        public string Director { get; set; }
        public string Writer { get; set; }
        public string Actors { get; set; }
        public bool Seen { get; set; }
        public string LocalPath { get; set; }
        public string Poster { get; set; }
    }
}